### PR TITLE
solution

### PR DESCRIPTION
**Pull request by abcdesign1**
[DEMO LINK](https://vadym4che.github.io/react_goods-selector/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -15,71 +19,97 @@ export const goods = [
...
-              className="button"
-            >
-              +
-            </button>
-          </td>
+    return (
+      <main className="section container">
+        <h1 className={
+          selectedGood ? 'title is-flex is-align-items-center' : 'title'
+        }
```
Comment (sergii-nosachenko) [inline src/App.tsx:46]: Use `classnames` library here

File: src/App.tsx [hunk 2]
```diff
@@ -15,71 +19,97 @@ export const goods = [
...
+          {selectedGood
+            ? `${selectedGood} is selected`
+            : 'No goods selected'}
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+          {selectedGood
+            && (
```
Comment (sergii-nosachenko) [inline src/App.tsx:53]:
```suggestion
          {selectedGood && (
```

File: src/App.tsx [hunk 3]
```diff
@@ -15,71 +19,97 @@ export const goods = [
...
-              +
-            </button>
-          </td>
+              return (
+                <tr
+                  key={good}
+                  data-cy="Good"
+                  className={
+                    selected ? 'Good has-background-success-light' : 'Good'
+                  }
```
Comment (sergii-nosachenko) [inline src/App.tsx:76]: Use `classnames` here

File: src/App.tsx [hunk 4]
```diff
@@ -15,71 +19,97 @@ export const goods = [
...
+                >
+                  <td>
+                    {selectedGood === good
```
Comment (sergii-nosachenko) [inline src/App.tsx:79]: Use `selected` constant here

File: src/App.tsx [hunk 5]
```diff
@@ -15,71 +19,97 @@ export const goods = [
...
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    selectedGood: 'Jam',
+  };
+
+  handleAddButton = (good: string) => {
```
[the remark this answers is not in the document]
Comment (vadym4che) [reply src/App.tsx:27]: fixed

File: src/App.tsx [hunk 6]
```diff
@@ -15,71 +19,97 @@ export const goods = [
...
+    this.setState({
+      selectedGood: good,
+    });
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  handleRemoveButton = () => {
```
[the remark this answers is not in the document]
Comment (vadym4che) [reply src/App.tsx:33]: fixed

File: src/App.tsx (hunk 2 above)
Comment (vadym4che) [reply src/App.tsx:53]: fixed - thank you for suggestions

File: src/App.tsx (hunk 3 above)
Comment (vadym4che) [reply src/App.tsx:76]: installed, imported and used

File: src/App.tsx (hunk 4 above)
Comment (vadym4che) [reply src/App.tsx:79]: fixed

**Pull request closed without merging**